docs/source/sophronia.rst

```diff
@@ -0,0 +1,251 @@
+       charge per peak and per slice for the SiPM to be considered.
+
+
+.. _Build event summary:
+
+Build event summary
+:::::::::::::::::::
+
+This part of the processing is equivalent to :doc:`dorothea` using
+``global_reco_algo`` and ``global_reco_params``. See that city's
+documentation for more details.
+
+.. _Resampling and hit creation:
+
+Resampling and hit creation
+:::::::::::::::::::::::::::
+
+In numerous ocasions, the energy deposition in a 1-:math:`\mu`\ s
+sample is not enough to produce a significant signal in the SiPMs,
+which results in poor reconstruction. It is therefore useful to be
+able to resample the waveforms and increase the sampling period (lower
+sampling rate). The variable that controls this resampling is
+``rebin`` and it specifies how many consecutive 1-:math:`\mu`\ s
+slices are added up together. A value of ``rebin = 1`` means no
+resampling is performed. Only integer values greater than or equal to
+1 are accepted.
+
+Unlike *Penthesilea*, *Sophronia* considers the response of a SiPM in
+a single (resampled) slice a *Hit*. Hits are defined as the
+aggregation of the absolute position of the SiPM, the time difference
+between the slice and the S1 peak time [2]_, the amplitude of the SiPM
+waveform (charge) and the energy corresponding to said charge. The
+corresponding energy is defined as
+
+.. math::
+
+   E_i = \frac{Q_i}{\sum_{k=0}^{N} Q_{k}} E_{slice}
+
+where :math:`Q_i` and :math:`E_i` are the charge and corresponding
+energy of SiPM :math:`i`, respectivly; :math:`N` is the number of
+SiPMs with signal in the slice and :math:`E_{slice}` is the energy of
+the slice, given by the amplitude of the PMT-summed waveform in the
+slice.
+
+.. [2] This information is used to obtain the z position of the hit by
+       dividing the time difference by the drift velocity
+       (``drift_v``).
+
+
+.. _Hit merging:
+
+Hit merging
+:::::::::::
+
+In some occasions, a time slice might have no SiPMs recorded. This
+means that all SiPM waveforms have been disregarded in :doc:`irene` or
+that no SiPM has a charge above ``q_thr`` in such slice. However,
+because the slice is considered part of a peak, there is some energy
+associated to it, which should not be discarded. In these cases,
+*Sophronia* generates a fake hit (a.k.a. *NN-hits*) with no charge and
+no position associated [3]_.
+
+After all hits are generated, a second step is performed, in which the
+fake hits are merged with existing valid hits. This operation searches
+for the hits that are closest (in z) to the fake one. Each fake hit's
+energy is shared among its neighbours and added to the neighbouring
+slice energy. The energy is not share homogeneously, but
+proportionally to the charge of each hit. The parameter ``same_peak``
+controls whether only slices from the same peak are taken into account
+or if other peaks might also be taken into consideration.
+
+.. [3] Actually, the algorithm associates an unphysical number to the
```

Comment:
Is `0` unphysical for a position? It's not for a coordinate. What do you mean that a position (3d-vector) has value equal to `0`?

Reply:
Poorly explained. It's not unphysical for the position, only for the charge.